src/server.tsx

```diff
@@ -5,33 +5,48 @@ import React from 'react';
 import { renderToStaticMarkup } from 'react-dom/server';
 import { extractCritical } from 'emotion-server';
 import { renderHtmlDocument } from './utils/renderHtmlDocument';
+import { fetchDefaultEpicContent, DefaultEpicContent } from './utils/fetchDefaultEpicContent';
 import { DefaultEpic } from './components/DefaultEpic';
 import cors from 'cors';
 
-const app = express();
-app.use(express.json({ limit: '50mb' }));
+const bootApp = (content: DefaultEpicContent): void => {
```

Comment:
What was the thinking behind not using top-level await in the end:? (Just curious really.) I've seen suggestions that it makes modules a bit harder to reason about/can be surprising, but on the flip-side for the key start file this might not matter and it might help remove the bootApp wrapping and indentation?

Reply:
Good question! It was a combination of things. When I looked it up I was reminded that, as you mentioned here, there is some push-back on whether or not top level await makes sense. I don't think any of the arguments against it affect us right now, but felt like maybe it was setting a precedent and it could be easy to avoid us having the conversation in future. 

The other (somewhat major 😸) reason is that it's [not supported in TypeScript yet](https://github.com/microsoft/TypeScript/issues/25988).

Reply:
Ha okay, that last argument is pretty slam-dunk :).

It looks like they're adding it in the next version - https://devblogs.microsoft.com/typescript/announcing-typescript-3-8-beta/.